documentation/src/main/asciidoc/topics/config_examples/server_network_fallback.xml

```diff
@@ -1,6 +1,8 @@
+<!-- Includes multiple strategies that {brandname} Server tries in the
+     declared order until it finds a match. -->
 <interfaces>
   <interface name="public">
-    <match-host value="my_host_name"/>
+    <match-host value="my_host_*"/>
```

Comment:
same here :) 
I'm [over] thinking if we should have a more concrete example, like `.*\.infinispan\.my-company\.org`?

Reply:
let's keep it simple. I didn't think about the consequences when I changed the example to use regex and it wasn't great. probably better to just tell users that they can use regex and leave them to it.